daemon.c

```diff
@@ -1540,8 +1556,10 @@ void *receiveOOK(void *param) {
 					/* Let's do a little filtering here as well */
 					if(r.length >= hw->minrawlen && r.length <= hw->maxrawlen) {
 						receive_queue(r.pulses, r.length, plslen, hw->hwtype);
+						r.length = 0;
 					}
-					r.length = 0;
+					// Avoid that short footer pulses (X10, IMPULS, ...) interfere with long header /sync pulses
+					if(r.length>20)r.length = 0;	// Avoid a buffer reset and preserve potential Header / SYNC / Start pulses
```

Comment:
> In the past this problem was neglected, the first pulsetrain was discarded, the header from the second (third, forth, …) pulsetrain was interpreted as footer, and the last pulsetrain was discarded.

I'm just not sure if this is the right solution. I believe we should create some logic to properly detect those headers instead of just dumb resetting the counter.